src/index.ts

```diff
@@ -7,7 +7,7 @@ import { enableProdMode, provide } from '@angular/core';
 import {bootstrap} from '@angular/platform-browser-dynamic';
 import { ROUTER_PROVIDERS } from '@angular/router-deprecated';
 import { APP_BASE_HREF } from '@angular/common/index';
-import { RioSampleApp } from './containers/sample-app';
+import { RioSampleAppComponent } from './containers/sample-app';
```

Comment:
yes it does but we can override that https://angular.io/docs/ts/latest/guide/style-guide.html#!#style-02-03